.editorconfig

```diff
@@ -0,0 +1,35 @@
+# https://editorconfig.org/
+# https://marketplace.visualstudio.com/items?itemName=EditorConfig.EditorConfig
```

Comment:
I don't know how many people use VS Code, but for those who do, this extension can make life much easier :slightly_smiling_face: 

This is just a suggestion, if you don't like it just let me know and I'll delete this file.

Reply:
Thanks for educating be about this extension :+1: 

https://github.com/mattermost/mattermost-plugin-github/pull/345 will include a standardised `.editorconfig` that all Mattermost plugins use. Would you please remove this file?